packages/openssl/packaging

```diff
@@ -0,0 +1,11 @@
+# abort script on any command that exits with a non zero value
+set -e -x
+
+mkdir ${BOSH_INSTALL_TARGET}/openssl
+echo "Extracting openssl"
+tar xzf openssl/openssl-1.0.2j.tar.gz
```

Comment:
Given the intimate link with bosh stemcell (arp etc ...), i suggest testing on a very recent stemcell, and also on other iaas (cloudwatt and vcloud) to be sure.